nncf/openvino/graph/model_transformer.py

```diff
@@ -610,10 +640,16 @@ def _apply_bias_insertion_transformations(
             node_output_port = node.output(transformation.target_point.port_id)
             node_output_source_ports = node_output_port.get_target_inputs()
 
-            bias_const_node = opset.constant(transformation.bias_value, dtype=node.get_element_type().to_dtype())
+            bias_node_name = f"{node_name}/nncf_null_bias_"
```

Comment:
Changed constant name for the new bias creation here, @alexsu52, @l-bat, @daniil-lyakhov